packages/extra/monitoring/templates/grafana/grafana.yaml

```diff
@@ -68,6 +68,7 @@ spec:
     metadata:
       annotations:
         kubernetes.io/ingress.class: "{{ $ingress }}"
```

Comment:
you could bravely remove it - it is obsolete

Reply:
done